standup/auth0/models.py

```diff
@@ -0,0 +1,13 @@
+from django.conf import settings
```

Comment:
There's no migration for this, yet. It's not hooked up to anything. Feel free to skip it.